src/claimtrie.h

```diff
@@ -28,6 +34,25 @@
 
 uint256 getValueHash(COutPoint outPoint, int nHeightOfLastTakeover);
 
+// lower-case and normalize any input string name
+// see: https://unicode.org/reports/tr15/#Norm_Forms
+inline const std::string normalizeClaimName(const std::string& name)
+{
+    boost::locale::localization_backend_manager manager =
+        boost::locale::localization_backend_manager::global();
+    manager.select("icu");
+
+    boost::locale::generator curLocale(manager);
+    const std::locale utf8 = curLocale("en_US.UTF8");
```

Comment:
It appears that the `utf8` object is immutable. Can we move the construction of it out of our inline method? It seems we could just construct a static instance once and be done with it.